src/injectedScript.ts

```diff
@@ -52,7 +53,7 @@ async function storeMessageFromSporran(
 
 async function startSession(
   unsafeDAppName: string,
-  dAppEncryptionKeyId: DidResourceUri,
+  dAppEncryptionKeyId: DidUrl,
```

Comment:
The naming `keyId` reflects the old naming scheme. At the very least, I would change it to `keyUri`.

Reply:
I'm not concerned with changing any of the naming since the purpose of this branch is just to see how the SDK alpha works with Sporran.

There are also a number of places where changing the naming would break other apps that interact with the Sporran API.

Reply:
Ok so these comments will be gone? I think it makes sense to leave the ones that would break otherwise, and change the ones that don't break.

Reply:
I'm not merging this branch

Reply:
Ok, it wasn't clear. I was asked to review, and I reviewed. Feel free to ignore most of the comments then.

Reply:
And I don't agree that having naming inconsistencies makes sense